packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -5056,7 +5056,6 @@ class RenderIndexedSemantics extends RenderProxyBox {
   @override
   void describeSemanticsConfiguration(SemanticsConfiguration config) {
     super.describeSemanticsConfiguration(config);
-    config.isSemanticBoundary = true;
```

Comment:
How does this not break things?

Reply:
the scrollable will be the semantics boundary. In its assemble semantics node, it still create a separate semantics node for every of its children. So the isSemanticsBoundary in RenderIndexSemantics is just performance improvement